webapp/packages/plugin-administration/src/locales/en.ts

```diff
@@ -85,4 +85,7 @@ export default [
 
   ['administration_disabled_drivers_title', 'Disabled drivers'],
   ['administration_disabled_drivers_search_placeholder', 'Search for the driver...'],
+  ['administration_disabled_drivers_enable_unsafe_driver_title', 'Enable unsafe driver'],
+  ['administration_disabled_drivers_enable_unsafe_driver_message', 'You are about to enable unsafe "{arg:driverName}" driver. Are you sure?'],
```

Comment:
```
Enabling this database driver may allow access to files on the server where this application is running. This could potentially expose sensitive system files or other protected data.

Only proceed if you fully understand the implications and trust the database configuration. Unauthorized or improper use of this driver may lead to security risks.

Do you want to enable the driver?
```